cmd/system-probe/api/server.go

```diff
@@ -46,6 +49,11 @@ func StartServer(cfg *config.Config) error {
 
 	mux.Handle("/debug/vars", http.DefaultServeMux)
 
+	// Expose telemetry endpoint
+	if ddconfig.Datadog.GetBool("telemetry.enabled") {
+		http.Handle("/telemetry", telemetry.Handler())
```

Comment:
Should this be `mux.Handle`?

Reply:
Could we move this to `app/run.go` inside the `isValidPort` check? It looks like a mistake here and I worry about it accidentally getting changed in the future. We also only want to turn add this handler if the debug HTTP server is running.